Pass variable_list of inputs to _wrap_outputs

malvika2147 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#23037 Pass variable_list of inputs to _wrap_outputs**

Summary: We don't need to convert the inputs from PyObject to a variable_list again in _wrap_outputs, since this is already done in process_outputs.

Differential Revision: [D16380071](https://our.internmc.facebook.com/intern/diff/D16380071)

Commit:
Pass variable_list of inputs to _wrap_outputs

Summary:

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

ezyang commented on torch/csrc/autograd/custom_function.cpp:
```diff
@@ -3,11 +3,18 @@
   const at::ArrayRef<Variable> raw_outputs,
   const std::shared_ptr<Function> &cdata) {
+
+std::unordered_set<at::TensorImpl*> inputs;
```
Nit: missing a single space of indent here

Commit:
Update on "Pass variable_list of inputs to _wrap_outputs"

Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#23037 Pass variable_list of inputs to _wrap_outputs**

Summary: We don't need to convert the inputs from PyObject to a variable_list again in _wrap_outputs, since this is already done in process_outputs.
